concourse-ci/README.md

```diff
@@ -1,9 +1,9 @@
 # Setup Concourse CI with Bosh
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcerdmann-pivotal%2Fazure-quickstart-templates%2Fconcourse-update%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
-<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fcerdmann-pivotal%2Fazure-quickstart-templates%2Fconcourse-update%2Fconcourse-ci%2Fazuredeploy.json" target="_blank">
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
```

Comment:
@cerdmann-pivotal 
I understand that you changed the link temporally for testing. But it should not be in the PR.